src/js/api/setup-steps.js

```diff
@@ -38,7 +38,8 @@ function setPlaylistAttributes(model, playlist, feedData) {
 
 function loadProvider(_model) {
     return loadPlaylist(_model).then(() => {
-        if (destroyed(_model)) {
+        const adConfig = _model.get('advertising');
+        if (destroyed(_model) || (adConfig && adConfig.outstream)) {
```

Comment:
Do we even need the loadProvider() in the setup promises for outsream? Could we remove it here to get around this check?
https://github.com/jwplayer/jwplayer-commercial/blob/v8.5.2/src/js/api/setup-steps.js#L49